common-libs/user-management/src/CognitoIdentityService.ts

```diff
@@ -1,3 +1,4 @@
+const createHash = require('create-hash/browser')
```

Comment:
why we use "require" here instead of "import"?

Reply:
current library do not support types for a browser module . i ll make a separate jira to find a better solution